### PR TITLE
Touvron2020training 1

### PR DESCRIPTION
**Pull request by specfazhou**
It is my draft summary for my last summary homework

### CODE REVIEW

File: _summaries/touvron2020training_1.md [hunk 1]
```diff
@@ -1,19 +1,47 @@
...
+
+<img width = 600 alt = "training_Deit_1" src = "training_Deit_1.png">
+
+* How is it realized (technically)? <br/>
+First, the distillation method requires one image classification model to be a teacher, and simply add one distillation token so that it can interact with class token and patch tokens in self-attention layers. As shown in the following graph <br/>
+<image width = 300 alt = "training_Deit_2" src = "training_Deit_2.png">
+
+There are actually two type of distillation methods -- soft distillation and hard distillation. The main difference between them is they use different objective functions.
+  In general, denote $Z_s$ is logits of student model, $Z_t$ is logits of teacher model, $KL$ is Kullback-Leibler divergence loss, $L_{CE}$ is cross-entropy. $\phi$ is softmax function, and $y_{t} = argmax_{c} Z_{t} (c)$ <br/>
```
Comment (rll2396) [inline _summaries/touvron2020training_1.md:21]: math mode requires double $

File: _summaries/touvron2020training_1.md [hunk 2]
```diff
@@ -1,19 +1,47 @@
...
-* How well does the paper perform?
-* What interesting variants are explored?
+* What is the core idea?  <br/>
+Vision Transformer can achieved excellent results on image classification, but it requires large training datasets and computationally expensive. The authors of this paper came up with a data-efficient image transformer (DeiT) that use ImageNet as training datasets, which has same architecture as Vision Transformer, but by using different training methods and distillation, DeiT has better performance even than EfficientNet (see the following graph). In addition, DeiT is significantly less computationally expensive than Vision Transformer as well. 
+
+<img width = 600 alt = "training_Deit_1" src = "training_Deit_1.png">
+
+* How is it realized (technically)? <br/>
+First, the distillation method requires one image classification model to be a teacher, and simply add one distillation token so that it can interact with class token and patch tokens in self-attention layers. As shown in the following graph <br/>
+<image width = 300 alt = "training_Deit_2" src = "training_Deit_2.png">
```
Comment (rll2396) [inline _summaries/touvron2020training_1.md:18]: This image doesn't make sense before their hard-label distillation method is explained. 

Also consider briefly explaining vision transformers, if only for completeness. I had no idea what the patch and class tokens are. The previous paper summary will probably cover this, so you could skip this when you actually present.

File: _summaries/touvron2020training_1.md [hunk 3]
```diff
@@ -1,19 +1,47 @@
...
+
+There are actually two type of distillation methods -- soft distillation and hard distillation. The main difference between them is they use different objective functions.
```
Comment (rll2396) [inline _summaries/touvron2020training_1.md:20]: Equations are hard to read, so include a high level interpretation. One such interpretation could be that soft means the student should try to mimic all of the teacher's probabilities, including those of the wrong classes, while hard means the student should only try to mimic the teacher's answer.

File: _summaries/touvron2020training_1.md [hunk 4]
```diff
@@ -1,19 +1,47 @@
...
+  1. Choosing CNNs as teacher model, the trained DeiT perform better than choosing Transformer as teacher model. <br/>
+  2. using hard distillation method helps DeiT achieve better performance <br/>
+  3. On other datasets, DeiT achieved state-of-the_art results or close to state-of-the-art results <br/>
+  4. data augmentation helps improve performance <br/>
+  5. AdamW optimizer with same learning as used in Vision Transformer helps DeiT achieved the best result. In general, DieT is sensitive for hyperparameters of optimizer. <br/>
+  6. higher fine-tuning resolution helps DeiT achieved better performance <br/>
+  
+  
+* What interesting variants are explored?<br/>
+  I don't think this paper mentions any variant of DeiT.
```
Comment (rll2396) [inline _summaries/touvron2020training_1.md:39]: Could summarize the ablations their ablation section here.

File: _summaries/touvron2020training_1.md (hunk 2 above)
Comment (specfazhou) [reply _summaries/touvron2020training_1.md:18]: That's good idea, I think the presentation about vision transformer will cover these things. Actually, I guess the patch tokens represent the image after embedding, but I am not completely sure, I didn't explain it.

File: _summaries/touvron2020training_1.md (hunk 3 above)
Comment (specfazhou) [reply _summaries/touvron2020training_1.md:20]: Thanks, that's very helpful. Already added them into my summary

File: _summaries/touvron2020training_1.md (hunk 4 above)
Comment (specfazhou) [reply _summaries/touvron2020training_1.md:39]: Thanks, I added it! your comments are very helpful to me
